database/migrationsPG/20191217191729_create-users-table-pg.js

```diff
@@ -59,8 +59,8 @@ exports.up = function (knex, Promise) {
       jobs.integer("carId").notNullable();
       jobs.string("address").notNullable();
       jobs.string("address2");
```

Comment:
Thank you for increasing the number of decimal places from 2 to 6 for the lat and lon coordinates.